__tests__/app.test.js

```diff
@@ -24,30 +39,73 @@ describe("Topics API", () => {
 				});
 		});
 	});
-});
-describe("Endpoints API", () => {
-	describe("Get Request", () => {
-		test("endpoints.json formatting correct", () => {
-			expect(endpoints).toBeInstanceOf(Object);
+	describe("Articles Endpoint", () => {
+		test("Get: 200, should return an array of all articles", () => {
```

Comment:
I think it's fine that you've got the toBeSortedBy expect statement in this test, but I would just reflect this in your test description 👍